AutoCollection/Performance.ts

```diff
@@ -208,10 +163,6 @@ class AutoCollectPerformance {
             this._client.trackMetric(combinedName + "nice", totalNice / combinedTotal);
```

Comment:
These metric names have to be verified for correctness as the legacy perfcounters are removed with these changes.